crates/hue/src/api/light.rs

```diff
@@ -501,7 +501,6 @@ pub enum LightEffect {
     Cosmos,
     Sunbeam,
     Enchant,
-    Sunrise,
```

Comment:
I guess it doesn't cause any problems to remove this effect?

Was it supposed to be a timed effect the whole time?

Reply:
Yes, I don't think it makes sense to use sunrise/sunset as an LightEffect as it doesn't repeat like the other "LightEffect"s. 
I added it there before I knew about the timed effect field.

Reply:
That's fine :+1: 

It also answers another question I had in the back of my mind: Should LightEffect and EffectType (from the zigbee impl) be merged, since they used to be identical.

The answer now is clearly: no. :)